src/app/components/doughnut-graph/doughnut-graph.component.ts

```diff
@@ -131,5 +113,30 @@ export class DoughnutGraphComponent implements OnInit {
     }
   }
 
+  getStatusMayastor(tests) {
+    try {
+      const passedCount = tests.filter((res) => res.status === 'PASSED').length;
+      const totalTestsCount = tests.length;
+      const percentage = (passedCount / totalTestsCount) * 100;
+      if (passedCount === 0) {
+        return `0 0`;
+      }
+      return `${percentage} ${100 - percentage}`;
+    } catch (error) {
+      console.log('\t Error', error);
+      return `100 0`;
```

Comment:
Why on error we return 100 ?

Reply:
its doughnut radius fill metrics . if data not present then doughnut will be on red color